coast/CLIMATOLOGY.py

```diff
@@ -76,10 +84,122 @@ def make_climatology(
                 ds = ds.drop_vars("clim_mean_ones_tmp")
 
         if fn_out is not None:
-            print("Saving to file. May take some time..")
+            info("Saving to file. May take some time..")
             with ProgressBar():
                 ds_mean.to_netcdf(fn_out)
 
         return ds_mean
 
-        return
+    @staticmethod
+    def _get_date_ranges(years, month_periods):
```

Comment:
Can we add type hints for the inputs and output here?

Reply:
Has now been updated